.chloggen/go-1.21.0.yaml

```diff
@@ -0,0 +1,13 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Please clarify how this is a breaking change. Is this a breaking change for the api changelog only?

Reply:
This is a breaking change for anyone importing collector-contrib modules, since the minimum version is now required in `go.mod`.